bucket-production.yml

```diff
@@ -1,5 +1,5 @@
 include:
-  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/v2.29.1/bucket-quality.yml'
+  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/v.2.29.2/bucket-quality.yml'
```

Comment:
v.2 instead of v2

Reply:
I forgot to bump, i also fixed this issue